src/DependencyInjection/Configuration.php

```diff
@@ -112,9 +112,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ])
                 ->prototype('scalar')->end()
             ->end()
-            ->scalarNode('slugify_service')
```

Comment:
To me, there is no point on doing this kind of thing, wdyt @VincentLanglet ? Symfony already provides decoration to change services. And since the slugifier is only used on the page manager, you can also override that definition.

Reply:
Yeah i think the same, and I guess it's the same reason that we have service class as parameter

Like

`%snapshot.manager.class%`

Reply:
I agree ; but if we want to do this the good way, we have to deprecate it on 3.x

Reply:
https://github.com/sonata-project/SonataPageBundle/pull/1511

Reply:
So, this one is done.